### ISSUE TITLE
External links and accessibility

### ISSUE THREAD

**Issue by jkshapiro** [gctools-outilsgc/design-system-code#241]
The page https://design.gccollab.ca/component/links/ includes the following guidance:
> # External Links
> If a link is bringing the user to an external web page, this should be indicated using an icon and the page should open in a new window.

It's worth mentioning that opening links in a new window isn't the greatest idea in the first place. Some relevant items from WCAG2.0:
* [Open new windows and tabs from a link only when necessary](https://www.w3.org/TR/WCAG20-TECHS/G200.html)
* [Give users advanced warning when opening a new window](https://www.w3.org/TR/WCAG20-TECHS/G201.html)&mdash;you need actual text, e.g. "opens in a new window", not just an icon
* As a Level AAA recommendation, [don't automatically open pages in a new window at all](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#consistent-behavior-no-extreme-changes-context).

**Comment by s-duffey** [comment]
@ethanWallace @piet0024 any thoughts on this? I just noticed now it says window rather than tab, and I think opening in a new tab is preferred, but based on the documentation above what should we decide?

**Comment by jkshapiro** [comment]
Window vs. tab isn't a material difference; they're both new contexts.

**Comment by s-duffey** [comment]
I think we fall under this category for the OADW applications: 

"However there are some situations where it is preferable from an accessibility perspective to open a new window or tab. Here are two such situations:... 2. The user is logged into a secured area of a site, and following a link to a page outside of the secured area would terminate the user's logon. In this case opening external links in an external window allows the user to access such references while keeping their login active in the original window."

However I think the point about including a warning is a good consideration. https://www.w3.org/TR/WCAG20-TECHS/G201.html
https://www.webcredible.com/blog/beware-opening-links-new-window/
